examples/css2d_label.html

```diff
@@ -39,7 +39,7 @@
 				var EARTH_RADIUS = 1;
 				var MOON_RADIUS = 0.27;
 
-				camera = new THREE.PerspectiveCamera( 45, window.innerWidth / window.innerHeight, 1, 1000 );
+				camera = new THREE.PerspectiveCamera( 45, window.innerWidth / window.innerHeight, 0.1, 200 );
```

Comment:
I would prefer to avoid small values of the near plane in the examples.

When the far plane is 200, I would expect a near plane of 1 would be fine.

Reply:
Yes, these values can still be improved. I think it make sense to look over the examples in general and optimize the near and far planes.